src/lib.rs

```diff
@@ -30,6 +53,12 @@ pub enum ByteCategory {
     NonAscii,
 }
 
+#[derive(Copy, Clone)]
+pub enum CharTable {
+    AsciiOnly,
+    CP437,
+}
```

Comment:
Since this is part of the public API, let's mark it [`non_exhaustive`](https://doc.rust-lang.org/reference/attributes/type_system.html#the-non_exhaustive-attribute) such that we can add more variants in the future without introducing breaking changes (technically).